src/components/lib/crt/crt.h

```diff
@@ -69,6 +73,12 @@ struct crt_comp {
 	coreid_t init_core;
 
 	crt_refcnt_t refcnt;
+
+	size_t tot_sz_mem;
+	size_t ro_sz;
+	struct crt_sinv *sinvs[CRT_COMP_SINVS_LEN];
```

Comment:
see comment above. Perhaps this shouldn't be pointers.

Reply:
ok, makes sense. I moved this to above the definition for crt_comp because of the issue with circular definition and I think that works

Reply:
Does that mean that it is no longer allocated within the component?